packages/commonwealth/client/scripts/views/pages/Community/Groups/Update/UpdateCommunityGroupPage.tsx

```diff
@@ -63,7 +64,10 @@ const UpdateCommunityGroupPage = ({ groupId }: { groupId: string }) => {
                 (y) => y.value === x.data.source.source_type,
               )?.label,
             },
-            requirementAmount: x.data.threshold,
+            requirementAmount: convertRequirementAmountFromWeiToTokens(
+              x.data.source.source_type,
+              x.data.threshold.trim(),
+            ),
```

Comment:
Can we use full variable names instead of `x` and `y`? Makes it harder to understand the code unless I scroll to the variable declarations.

Reply:
Done